src/main/java/me/flashyreese/mods/sodiumextra/mixin/features/cloud/MixinWorldRenderer.java

```diff
@@ -108,11 +110,12 @@ private void renderClouds(BufferBuilder builder, double x, double y, double z, V
 
     }
 
-    @Redirect(
-            method = "renderClouds(Lnet/minecraft/client/util/math/MatrixStack;FDDD)V",
-            at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/SkyProperties;getCloudsHeight()F")
-    )
-    private float getCloudHeight(SkyProperties skyProperties) {
-        return SodiumExtraClientMod.options().extraSettings.cloudHeight;
+    @ModifyVariable(method = "renderClouds", index = 18, at = @At(value = "STORE"))
```

Comment:
I don't like how this fix is done, perhaps you can check whether SkyProperties' `cloudHeight` is `Float.NaN`.